packages/runtime/container-runtime/src/gc/gcEarlyAdoption.md

```diff
@@ -22,9 +22,14 @@ The eventName for the Inactive log ends with `InactiveObject_Loaded`.
 
 The default timeout for an unreferenced object to become "Inactive" is 7 days. This is intended to be long enough such that
 it's very unlikely to hit a legitimate case where an object is revived within the same session it was deleted (e.g. delete then undo).
-Based on your application's user experience, you may choose to shorten this timeout to get an earlier signal (but beware of false positives).
+Based on your application's user experience, you may choose to shorten this timeout to get an earlier signal.
 
-To override the default InactiveObject timeout, use the `inactiveTimeoutMs` GC Option.
+This option comes with a big caveat though: The design we've chosen for GC means that until GC runs, the system has incomplete knowledge
```

Comment:
I don't believe we should mention using inactive telemetry in our docs at all. We should move it to internal docs for use by internal customers.
If we do, it will get used and users would run into issues.